src/test/java/uk/gov/hmcts/reform/sendletter/SampleData.java

```diff
@@ -32,6 +37,56 @@ public static String uuid() {
         return UUID.randomUUID().toString();
     }
 
+    public static MigrationInfo flywayInfo(MigrationState state) {
+        return new MigrationInfo() {
+
+            @Override
+            public MigrationType getType() {
+                return MigrationType.SQL;
+            }
+
+            @Override
+            public Integer getChecksum() {
+                return null;
+            }
+
+            @Override
+            public MigrationVersion getVersion() {
+                return null;
+            }
+
+            @Override
+            public String getDescription() {
+                return null;
+            }
+
+            @Override
+            public String getScript() {
+                return "some-script.sql";
+            }
+
+            @Override
+            public MigrationState getState() {
+                return state;
+            }
+
+            @Override
+            public Date getInstalledOn() {
+                return null;
+            }
+
+            @Override
+            public Integer getExecutionTime() {
+                return null;
+            }
+
+            @Override
+            public int compareTo(MigrationInfo o) {
+                return 0;
+            }
+        };
+    }
```

Comment:
can we get away with just using a mock in tests?

Reply:
was thinking of using mocks but got all lazy implementing 3 or 4 mocks returning migration info

Reply:
not sure why it looked more hassle